src/movie/Home.js

```diff
@@ -1,10 +1,12 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router';
 import { MovieSearchBanner, MovieList, MovieDetails } from '.';
 
-export default class Home extends Component {
-  constructor() {
+class Home extends Component {
+  constructor(props) {
     super();
```

Comment:
let's add **props** as argument for **super** method

https://medium.com/@jbbpatel94/why-do-we-write-super-props-fc367074a2af